opentutor_classifier/tests/test_cli.py

```diff
@@ -98,7 +106,7 @@ def test_cli_outputs_models_files(tmpdir, lesson, no_of_expectations, shared_roo
             "peer pressure can change your behavior",
             ARCH_DEFAULT,
             [
-                _TestExpectation(expectation="0", score=0.8, evaluation="Good"),
+                _TestExpectation(expectation="0", score=0.72, evaluation="Good"),
```

Comment:
these look like biggish drops in confidence...are we sure this is OK? Is something just weird/wrong w cli version?

Reply:
It's weird for me too..because it is related to the Default classifier...and it has not been changed. Let me look at this one more time.